openadapt/start.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
```

Comment:
Can you please add a module docstring, with example usage, as per Google Py Doc Style?

Reply:
Addressed in [b961eab](https://github.com/MLDSAI/OpenAdapt/pull/201/commits/b961eabeeaa1822ab114b2edd4524f888624f6f4)